test/OpenTelemetry.Instrumentation.AWS.Tests/TestAWSClientInstrumentation.cs

```diff
@@ -202,6 +210,253 @@ public async Task TestSQSSendMessageSuccessful()
+                   .AddInMemoryExporter(exportedItems)
+                   .Build())
+        {
+            var bedrock = new AmazonBedrockClient(new AnonymousAWSCredentials(), RegionEndpoint.USEast1);
+            string requestId = @"fakerequ-esti-dfak-ereq-uestidfakere";
+            string dummyResponse = "{\"GuardrailId\":\"123456789\"}";
+            CustomResponses.SetResponse(bedrock, dummyResponse, requestId, true);
+            var get_guardrail_req = new GetGuardrailRequest();
+            get_guardrail_req.GuardrailIdentifier = "123456789";
+#if NETFRAMEWORK
+            var response = bedrock.GetGuardrail(get_guardrail_req);
+#else
+            var response = await bedrock.GetGuardrailAsync(get_guardrail_req);
+#endif
+            Assert.Single(exportedItems);
+            Activity awssdk_activity = exportedItems[0];
+
+            this.ValidateAWSActivity(awssdk_activity, parent);
+            this.ValidateBedrockActivityTags(awssdk_activity);
+
+            Assert.Equal(Status.Unset, awssdk_activity.GetStatus());
+            Assert.Equal(requestId, Utils.GetTagValue(awssdk_activity, "aws.requestId"));
+        }
+    }
+
+    [Fact]
+#if NETFRAMEWORK
+    public void TestBedrockRuntimeInvokeModelSuccessful()
+#else
+    public async Task TestBedrockRuntimeInvokeModelSuccessful()
+#endif
+    {
+        var exportedItems = new List<Activity>();
+
+        var parent = new Activity("parent").Start();
+
+        using (Sdk.CreateTracerProviderBuilder()
+                   .AddXRayTraceId()
+                   .SetSampler(new AlwaysOnSampler())
+                   .AddAWSInstrumentation()
+                   .AddInMemoryExporter(exportedItems)
+                   .Build())
+        {
+            var bedrockruntime = new AmazonBedrockRuntimeClient(new AnonymousAWSCredentials(), RegionEndpoint.USEast1);
+            string requestId = @"fakerequ-esti-dfak-ereq-uestidfakere";
+            string dummyResponse = "{}";
+            CustomResponses.SetResponse(bedrockruntime, dummyResponse, requestId, true);
+            var invoke_model_req = new InvokeModelRequest();
+            invoke_model_req.ModelId = "amazon.titan-text-express-v1";
+#if NETFRAMEWORK
+            var response = bedrockruntime.InvokeModel(invoke_model_req);
+#else
+            var response = await bedrockruntime.InvokeModelAsync(invoke_model_req);
+#endif
+            Assert.Single(exportedItems);
+            Activity awssdk_activity = exportedItems[0];
+
+            this.ValidateAWSActivity(awssdk_activity, parent);
+            this.ValidateBedrockRuntimeActivityTags(awssdk_activity);
+
+            Assert.Equal(Status.Unset, awssdk_activity.GetStatus());
+            Assert.Equal(requestId, Utils.GetTagValue(awssdk_activity, "aws.requestId"));
+        }
+    }
+
+    [Fact]
+#if NETFRAMEWORK
+    public void TestBedrockAgentRuntimeInvokeAgentSuccessful()
+#else
+    public async Task TestBedrockAgentRuntimeInvokeAgentSuccessful()
```

Comment:
Could you please also add a test case to cover knowledgebaseId on BedrockAgentRuntime?